backend/src/main/java/com/google/rolecall/jsonobjects/UserInfo.java

```diff
@@ -157,6 +170,18 @@ public static Builder newBuilder() {
     @JsonProperty("manageRules")
     public abstract Builder setManageRules(Boolean manageRules);
 
+    @JsonProperty("emergencyContcatName")
+    public abstract Builder setEmergencyContactName(String emergencyContcatName);
+
+    @JsonProperty("emergencyContactName")
+    public abstract Builder setEmergencyContactNumber(String emergencyContactNumber);
```

Comment:
Just FYI, I think in the frontend we do not have field to display these 2 fields.

Reply:
As I am not very familiar with Typescript, I only reviewed the backend code and they look good to me.

Reply:
Zephyr, yes the emergency fields are not implemented in the user interface yet. The are other fields that are implemented on the backend but not implemented on the front end yet.

Reply:
Thanks for confirming that.